### DISCUSSION TITLE
Weasel or stoat?

### DISCUSSION THREAD

**Discussion by jorgeluismireles** [vlang/v#22774] [General]
Hi everybody. From the beginning I assumed the V mascot at https://vlang.io/ was a kind of gopher that went to the gym. I saw a [youtube video](https://www.youtube.com/watch?v=Ukzi7yBPbps) and make me wonder what exactly is. By the tail form, seems to be a stoat, or not?

![weasel_or_stoat](https://github.com/user-attachments/assets/91cbd09d-93e3-4b1b-a70f-f6a2d983800d)

**Comment by medvednikov** [comment]
Lol you're right. The tail is too long.

It's supposed to be a Veasel :)

Maybe it's a special weasel with a long tail :)
